source/administrator-guide/configuring-the-catalog/inspire-configuration.rst

```diff
@@ -30,87 +30,95 @@ When enabled, the INSPIRE support activate the following:
 Loading INSPIRE codelists
 -------------------------
 
-For metadata describing INSPIRE dataset and series, it is recommended to load the following codelist from the `INSPIRE Registry<http://inspire.ec.europa.eu/registry/>`_:
-
-* `INSPIRE Theme <http://inspire.ec.europa.eu/theme>`_
-* Metadata codelist register > `Spatial scope <http://inspire.ec.europa.eu/metadata-codelist/SpatialScope>`_
-* Metadata codelist register > `INSPIRE priority data set <http://inspire.ec.europa.eu/metadata-codelist/PriorityDataset>`_
-
-From ``admin console`` > ``classification systems``, user can load thesaurus from the registry:
+To describe INSPIRE datasets and series, it is recommended to load relevant codelists from the `INSPIRE Registry <http://inspire.ec.europa.eu/registry/>`_, 
+the following codelists are relevant in the scope of metadata guidelines v2.0:
+
+* `INSPIRE Theme <https://inspire.ec.europa.eu/theme>`_
+* `Application schema <https://inspire.ec.europa.eu/applicationschema>`_
+* `Media types <https://inspire.ec.europa.eu/media-types>`_
+* Metadata codelist register > `Protocols <https://inspire.ec.europa.eu/metadata-codelist/ProtocolValue>`_
+* Metadata codelist register > `Spatial scope <https://inspire.ec.europa.eu/metadata-codelist/SpatialScope>`_
+* Metadata codelist register > `INSPIRE priority data set <https://inspire.ec.europa.eu/metadata-codelist/PriorityDataset>`_
+* Metadata codelist register > `Spatial Data Service Category <https://inspire.ec.europa.eu/metadata-codelist/SpatialDataServiceCategory>`_
+* Metadata codelist register > `Conditions applying to Access and Use <https://inspire.ec.europa.eu/metadata-codelist/ConditionsApplyingToAccessAndUse>`_
+* Metadata codelist register > `Limitations on Public Access <https://inspire.ec.europa.eu/metadata-codelist/LimitationsOnPublicAccess>`_
+* Metadata codelist register > `OnLine Description Code <https://inspire.ec.europa.eu/metadata-codelist/OnLineDescriptionCode>`_
+* Metadata codelist register > `Quality of Service Criteria <https://inspire.ec.europa.eu/metadata-codelist/QualityOfServiceCriteria>`_
+ 
+
+From ``admin console`` > ``classification systems`` > ``Thesaurus``, administrators can manage thesauri. One of the options is to load a thesaurus straight from the registry. 
+It is important to verify the thesaurus date, because in some case INSPIRE requires a specific date of the thesaurus. After import you can download the thesaurus, change it
```

Comment:
If there is a bug, we should fix it no ? What thesaurus loaded from Registry needs a date to be updated ? We have this in place here https://github.com/geonetwork/core-geonetwork/blob/master/web/src/main/webapp/xslt/services/thesaurus/registry-to-skos.xsl#L77-L84 so it is only  a matter of configuration. User reading that does not really know what to do ? What is the correct date ?

Reply:
I understood that the problem is on the side of the eu registry, which is not able to store snapshots of a thesaurus by date. and will always return the current date.

Dates should be used as mentioned in TG

Yes, we could introduce a list in geonetwork to introduce the relevant date for each of the inspire thesauri, but i'd prefer this is solved at the registry (or in the guidelines, not to require a date)

Let me remove the section for now